pkg/variantregistry/ocp.go

```diff
@@ -245,7 +245,7 @@ var (
 	// We're not sure what these aggregator jobs are but they exist as of right now:
 	aggregatorRegex = regexp.MustCompile(`(?i)aggregator-`)
 	alibabaRegex    = regexp.MustCompile(`(?i)-alibaba`)
-	arm64Regex      = regexp.MustCompile(`(?i)-arm64|-multi-a-a`)
+	arm64Regex      = regexp.MustCompile(`(?i)-arm64|-multi-a-a|-arm`)
```

Comment:
Technically we could just remove `-arm64` in place of `-arm` but seems like the `-arm` job might be better off to be renamed at some point.